README.md

```diff
@@ -1,73 +1,16 @@
 # Amsterdam React Maps
+A repo with components / utilities to be used in projects for Amsterdam. 
```

Comment:
I wouldn't use 'Amsterdam', but perhaps refer to project that have to adhere to the Amsterdam style-guide or projects that published by Datapunt.